go/vimlparser.go

```diff
@@ -94,6 +94,7 @@ var NODE_REG = 89
 var NODE_CURLYNAMEPART = 90
 var NODE_CURLYNAMEEXPR = 91
 var NODE_LAMBDA = 92
+var NODE_PARENEXPR = 93
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
don't use ALL_CAPS in Go names; use CamelCase